sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/OptimizeSkewedJoin.scala

```diff
@@ -48,9 +48,6 @@ import org.apache.spark.sql.internal.SQLConf
  * (L2-1, R2), (L2-2, R2),
  * (L3, R3-1), (L3, R3-2),
  * (L4-1, R4-1), (L4-2, R4-1), (L4-1, R4-2), (L4-2, R4-2)
- *
- * Note that, when this rule is enabled, it also coalesces non-skewed partitions like
- * `CoalesceShufflePartitions` does.
```

Comment:
Is this comment not true anymore?

Reply:
no, the partitions will be coalesced by rule `CoalesceShufflePartitions` later.